.drone-local.yml

```diff
@@ -68,7 +69,7 @@ services:
 matrix:
   include:
     - {
-        GO: "golang:1.8", DATABASE: postgres, VERSION: 9.6, REFORM_DRIVER: postgres,
+        GO: "golang:1.9", DATABASE: postgres, VERSION: 9.6, REFORM_DRIVER: postgres,
```

Comment:
Maybe it is possible to use some pre-defined variables for this Go version to not to change a lot of similar lines each time?

Reply:
That file is a subset of .drone.yml. I just copy/paste a part of it. Making a variable will make it harder.